tests/test_qpex.py

```diff
@@ -311,11 +312,10 @@ def test_qp_ex_rc_bind_mw(self):
         client.qp.wr_rdma_write(new_key, server.mr.buf)
         client.qp.wr_set_sge(client_sge)
         client.qp.wr_complete()
-        try:
-            u.poll_cq(client.cq)
-        except PyverbsRDMAError as ex:
-            if ex.error_code != e.IBV_WC_REM_ACCESS_ERR:
-                raise ex
+        wcs = u._poll_cq(client.cq)
+        if wcs[0].status != e.IBV_WC_REM_ACCESS_ERR:
+            raise PyverbsRDMAError('Completion status is {s}'.
```

Comment:
it's preferable to use f-strings (i.e. f'Completion status is {wc_status_to_str(wc[0].status)}' )

Reply:
Sounds good